microsoft-edge/progressive-web-apps-chromium/demo-pwas.md

```diff
@@ -0,0 +1,139 @@
+---
+description: This page lists sample apps you can use to learn more about how to build PWAs.
+title: Sample PWAs
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/28/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, demos, samples
+---
+# Sample PWAs
+
+Use this list of sample apps to learn more about how to use PWA features and APIs.
+
+:::row:::
```

Comment:
@captainbrosset - are you *sure* 2-column gives a better UX / CX than simple, linear layout?

Linear layout:
- Pro: image is big and crisp and in-context

2-col:
- Con: tagging is hard to author and maintain.  Remember, a Dev or PM might need to change this section.
- Con: When you put window half-width, the images are tiny inline.

Reply:
Those are good points. I'll try another approach.

Reply:
I'll push a commit with 1 column instead of 2.
The goal for this page is to act as a quick way for readers to find actual source code that uses the features documented on the docs website. They can scroll through the page until they find an app that uses feature X, and then click to navigate to the source code and find a real usage example for feature X there.
So the images aren't that important. They're nice visually, and they might also make people realize that PWAs can be quite powerful, and nice looking, but they don't have to be so big. They're not helping follow a procedure or anything like that.

So, agreed on the 1-column layout, but I might want to try having smaller (thumbnail) images instead. I'll try this later.

Reply:
@captainbrosset - The strategy I have in mind for such a situation in a future PR is:

Keep the existing, big picture as the lightbox png, then copy/paste that png to create the inline png, then resize the inline png smaller, and rename it like "foo_reduced.png" to hint that the lightbox png is the original higher-res upstream source file.  Or equivalent naming scheme to relate the two versions of the png file.

Test with narrow & wide window.  Extra fun: start w/ narrow window, activate the lightbox, then drag/resize the window and watch lightbox do strange resizing.